<html lang="en-US">
<head>
    <meta charset='utf-8'>
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <link rel="stylesheet" href="/polonez-project/assets/css/style.css?v=9c959f14212481793cea53c43651dfb33341fbb7">
    <link rel="stylesheet" type="text/css" href="/polonez-project/assets/css/print.css" media="print">

	<title>
	My Polonez Fellowship - Tools for searching pathogenic variants in regulatory regions of the human genome
	</title>

</head>

<body>

<b>
Tools for searching pathogenic variants in regulatory regions of the human genome
</b>

<p>
Monogenic diabetes is a genetic disease responsible for approximately 5% of diabetes cases. 
It is known to be caused by mutations of one of several protein-coding genes, each of which defines a subtype of the disease (e.g. several types of MODY, neonatal diabetes, several types of complex genetic syndromes). 
Precise molecular diagnosis, such as identification of the disrupted gene, can have impact on the treatment and course of the disease. 
For instance, patients with the most common type of monogenic diabetes, HNF1A-MODY, caused by mutations in the HNF1A gene, can completely substitute insulin injections with sulfonylurea pills. 
Patients with mutations in glucokinase gene (GCK) causing GCK-MODY can often completely drop treatment. Among patients likely affected by monogenic diabetes, about 25-40% gets a molecular diagnosis. 
The remaining part shows no defects in the known disease-causing genes. 
The situation is similar for many other rare monogenic disorders. 
We hypothesize that the genetic factors responsible for the disease can hide outside of the protein-coding genes.
</p>

<p>
Human genome, besides the protein-coding genes, contains large number of regulatory regions and non-coding genes with regulatory function.
Researchers estimate that as much as 20-40% of the genome can have regulatory function, compared to only about 1% that encodes proteins. 
These regulatory elements can act in different ways effectively modulating the amount of gene products - proteins. 
Disrupted regulation results in abnormal amount of a functional protein, in contrast to a situation where a gene is mutated, which leads to a normal amount of a defective protein.
The final effect can be very similar, and numerous reports of disorders caused by disrupted gene regulators have been published in scientific literature. 
The challenge is that the regulatory regions of human genome are not as well characterized as protein-coding genes. 
In consequence, the effects of mutations in regulatory regions are much more difficult to assess. 
Considering the vast size of these regions, locating one causative mutation is like searching a needle in a haystack. 
So, even though sequencing an entire human genome has become readily affordable, making sense of the data remains a great challenge. 
</p>

<p>
During the course of this project entitled <i>“Development of novel bioinformatics software for analysis of genetic variation in non-coding elements of the human genome: case of a rare monogenic disorder.“</i>, we aim to tackle this challenge. 
We will develop a method that combines publicly available data on cell-type and tissue specific regulators in human genome, to facilitate searching regulatory regions relevant for an arbitrary monogenic disorder. 
The method will be implemented as a web application – ReMuS - to enable easy access for interested researchers working on monogenic phenotypes. 
</p>

<p>
The method will be tested on a subtype of monogenic diabetes, HNF1B-MODY, which presents with diabetes, renal cysts and/or additional features. 
We will collect DNA samples from patients with a clinical presentation of HNF1B-MODY, but carrying no mutations in the HNF1B gene.
Sequencing genomes of the patients and searching causative variants using ReMuS will help us to fine-tune the method and showcase its usefulness.
We believe that some of the patients involved in the project will learn about the molecular causes of their disease. 
We also hope that genetic findings of this project will tell us more about the mechanism of the disease, and that they will have translational potential, contributing to better diagnosis, management and treatment of HNF1B-MODY.
</p>

<p>
This project is financed by a Polonez 3 grant awarded by the National Science Center in Poland, funded by the European Union’s Horizon 2020 research and innovation programme under the Marie Skłodowska-Curie grant agreement.
</p>
</body>
</html>
